src/main/java/org/darisadesigns/polyglotlina/DictCore.java

```diff
@@ -321,32 +241,18 @@ public ConWordCollection getWordCollection() {
     /**
      * Builds a report on the conlang. Potentially very computationally
      * expensive.
+     * 
+     * @param progress
      */
-    public void buildLanguageReport() {
+    public void buildLanguageReport(PLanguageStatsProgress progress) {
```

Comment:
This is honestly some of the worst code in the entire project, and I know there are some messy areas... for now probably best to just handle it as is, but it's definitely something I've been eyeing to make better/replace for some time now, just as a heads up.